wcivf/apps/elections/templates/elections/includes/_cancelled_election.html

```diff
@@ -47,7 +47,7 @@ <h4>{% trans "Cancelled and Rescheduled Election" %}</h4>
 {% else %}
     {% if object.metadata.cancelled_election %}
         <h4>{{ object.metadata.cancelled_election.title }}</h4>
-        <p>{{ object.metadata.cancelled_election.detail }}</p>
+        {{ object.metadata.cancelled_election.detail | linebreaks }}
```

Comment:
Using the `| linebreaks` filter renders the whole thing in a `<p>` for us and interprets `\n\n` as starting a new `<p>` so that's why I've dropped the `<p>` wrapper here
https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#linebreaks